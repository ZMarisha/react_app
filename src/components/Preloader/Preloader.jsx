import React from "react";

const Preloader = () => {
    return (
        <div style={{
            position: 'absolute',
            width: '100%',
            height: '100%',
            background: 'white'
            }}>
            <img src='https://flevix.com/wp-content/uploads/2019/07/Color-Loading-2.gif' alt='preloader' style={{
                display: 'block',
                objectFit: 'contain',
                height: 'inherit',
                width: 'inherit',
            }}/>
        </div>
    )
}

export default Preloader;