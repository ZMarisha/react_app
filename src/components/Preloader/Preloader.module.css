.preloader {
    position: absolute;
    width: 100%;
    height: 100%;
    background: white;
}
.preloader img {
    display: block;
    object-fit: contain;
    height: inherit;
    width: inherit;
}