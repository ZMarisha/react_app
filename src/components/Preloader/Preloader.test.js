import { render } from "@testing-library/react";
import Preloader from "./Preloader";

test('Preloader styles', () => {
    expect(render(<Preloader />)).toMatchSnapshot()
})