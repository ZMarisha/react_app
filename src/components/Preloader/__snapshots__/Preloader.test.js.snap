// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preloader styles 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        style="position: absolute; width: 100%; height: 100%; background: white;"
      >
        <img
          alt="preloader"
          src="https://flevix.com/wp-content/uploads/2019/07/Color-Loading-2.gif"
          style="display: block; object-fit: contain; height: inherit; width: inherit;"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      style="position: absolute; width: 100%; height: 100%; background: white;"
    >
      <img
        alt="preloader"
        src="https://flevix.com/wp-content/uploads/2019/07/Color-Loading-2.gif"
        style="display: block; object-fit: contain; height: inherit; width: inherit;"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
